Enable random_channel_buffer_tests to run on Linux

kain88-de opened the pull request:
With this patch applied I can see the following line in the test output:

   test tests::random_channel_buffer_tests::test_random_channel_buffers has been running for over 60 seconds

Without it I run in a failure with 'Invalid keystroke `cmk-,`



Release Notes:

- N/A

maxdeviant commented on crates/collab/src/tests/test_server.rs:
```diff
@@ -271,6 +271,12 @@ impl TestServer {
+        let os_keymap = if cfg!(target_os = "linux") {
+            "keymaps/default-linux.json"
+        } else {
+            "keymaps/default-macos.json" // Default case if none of the above
```
I don't think we need this comment here:

```suggestion
            "keymaps/default-macos.json"
```

kain88-de committed:
Update crates/collab/src/tests/test_server.rs

Co-authored-by: Marshall Bowers <elliott.codes@gmail.com>

kain88-de committed:
Enable random_channel_buffer_tests to run on Linux

With this patch applied I can see the following line in the test output:

   test tests::random_channel_buffer_tests::test_random_channel_buffers has been running for over 60 seconds

Without it I run in a failure wit 'Invalid keystroke `cmk-,`

kain88-de commented:
When do you plan to run the unit tests on linux?

maxdeviant commented:
> When do you plan to run the unit tests on linux?

Probably as we get closer to a full Linux release.

I'm not sure what the current state of the tests is.

kain88-de commented:
Currently 38 tests fail with this patch applied. I assume most of those are similar issues. I can work a bit on fixing them. I would feel better about that if the CI runs the tests to ensure they are not broken again in the future. Ideally all new test would also pass on Linux. 

However, do you want this at the moment?

maxdeviant commented:
> Currently 38 tests fail with this patch applied. I assume most of those are similar issues. I can work a bit on fixing them. I would feel better about that if the CI runs the tests to ensure they are not broken again in the future. Ideally all new test would also pass on Linux.
> 
> However, do you want this at the moment?

I'm not sure that we're ready to enable Linux tests in CI quite yet.
